### DISCUSSION TITLE
我是艾伦•耶格尔今年19岁，我的好兄弟莱纳•布朗有一天走丢了如果你找到了他请把他带到帕拉迪岛

### DISCUSSION THREAD

**Discussion by nm114515** [share121/inter-knot#5289] [委托]
![Screenshot_20240922_120113_edit_90952408993497](https://github.com/user-attachments/assets/de551473-dec1-49ec-b6f7-77cd4159611e)

**Comment by LiuLiBHZ** [comment]
我见过这个人，他天天坐在椅子上不知道在想什么。

**Reply by nm114515** [reply]
他现在在哪

**Comment by LiuLiBHZ** [comment]
好像是什么马莱精神病院，天天念叨什么巨人啊，城门之类的。估计是疯了

**Reply by nm114515** [reply]
等等，他在马莱？马莱这个地方我在父亲的记忆里看见过
